examples/chart/teleport-kube-agent/templates/_config.tpl

```diff
@@ -128,6 +128,44 @@ discovery_service:
   enabled: false
 {{- end }}
 
+windows_desktop_service:
+  {{- if contains "windowsdesktop" (.Values.roles | toString) }}
+  enabled: true
+  {{- if not (or (.Values.windowsDesktopLDAP) (.Values.windowsDesktopNonADHosts) ) }}
+    {{- fail "at least one of 'windowsDesktopLDAP', 'windowsDesktopNonADHosts' is required in chart values when windowsdesktop role is enabled, see README" }}
```

Comment:
What if you want to register hosts via API instead of in the helm chart? That should be possible, no?

Reply:
Removed